frontend/src/metabase/ui/components/theme/ThemeProvider/ThemeProvider.tsx

```diff
@@ -73,6 +73,7 @@ export const ThemeProvider = (props: ThemeProviderProps) => {
       theme={theme}
       getStyleNonce={() => window.MetabaseNonce ?? "metabase"}
       classNamesPrefix="mb-mantine"
+      cssVariablesSelector=".mb-wrapper"
```

Comment:
Without this, we are targeting `:root` which overrides the customer's Mantine config.